frappe/auth.py

```diff
@@ -158,8 +158,10 @@ def get_user_info(self):
 		self.info = frappe.get_cached_value(
 			"User", self.user, ["user_type", "first_name", "last_name", "user_image"], as_dict=1
 		)
-
-		self.user_type = self.info.user_type
+		try:
+			self.user_type = self.info.user_type
+		except AttributeError:
+			pass
```

Comment:
without self.info it's gonna fail anyway, not point in handling it here. 

This is failure with extra steps.

Reply:
Go-http-client? That might not be users but our uptime checker.